pkgs/tools/virtualization/distrobuilder/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, pkgconfig, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn
+, makeWrapper, coreutils, gnupg, gnutar, squashfsTools}:
+
+buildGoPackage rec {
+  name = "distrobuilder-unstable-${version}";
+  version = "2018-04-28";
```

Comment:
I think we prefer to use either dashes or dots here. Otherwise upgrading doesn't work correctly.